scripts/run-with-strimzi.sh

```diff
@@ -28,16 +28,16 @@ function cleanTmpDir {
 }
 trap cleanTmpDir EXIT
 
-if [[ -z "${QUAY_ORG:-}" ]]; then
-  echo "Please set QUAY_ORG, exiting"
+if [[ -z "${REGISTRY_DESTINATION:-}" ]]; then
+  echo "Please set REGISTRY_DESTINATION to a value like 'quay.io/<myorg>/kroxylicious', exiting"
```

Comment:
Why do we exit here given we have a default destination? Unlike the old model of just `QUAY_ORG` we have a full destination as the default and could thus continue.

Reply:
Agreed.  I changes the logic to use the default if REGISTRY_DESTINATION.  I tested both ways locally. All seems good.